connector/src/main/java/org/geysermc/connector/utils/SkinUtils.java

```diff
@@ -120,22 +125,28 @@ public static void requestAndHandleSkinAndCape(PlayerEntity entity, GeyserSessio
                         SkinProvider.Skin skin = skinAndCape.getSkin();
                         SkinProvider.Cape cape = skinAndCape.getCape();
 
-                        if (cape.isFailed()) {
-                            cape = SkinProvider.getOrDefault(SkinProvider.requestBedrockCape(
-                                    entity.getUuid(), false
-                            ), SkinProvider.EMPTY_CAPE, 3);
-                        }
+                        SkinProvider.SkinGeometry geometry;
+                        if (!(entity instanceof SkullPlayerEntity)) {
```

Comment:
You should probably make a method specific for Skulls in the SkinProvider as well